media-libs/hamlib/files/hamlib-4.5.5-gcc14-fix.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/tests/memload.c b/tests/memload.c
```

Comment:
Sent upstream?

Reply:
No, and still not.

Reply:
@thesamesam Do we need some sort of required status check for modifications to files/* that the PR author acknowledges the patch has been sent upstream?

@listout